Closure_Project/rest_api/models.py

```diff
@@ -169,15 +186,15 @@ def __str__(self):
 
 class Student(models.Model):
     user = models.OneToOneField(User, on_delete=models.CASCADE)
-    track = models.ForeignKey(Track, on_delete=models.CASCADE)
-    year_in_studies = models.IntegerField(choices=Year.choices)
+    track = models.ForeignKey(Track, on_delete=models.CASCADE, null=True)
+    year_in_studies = models.IntegerField(choices=Year.choices, null=True)
     courses = models.ManyToManyField(Course, through='Take', blank=True)
 
     def __str__(self):
         return ', '.join((self.user.username,
                           self.user.get_full_name(),
                           f'year={self.year_in_studies}',
-                          f'track={self.track.track_number}',
+                          f'track={self.track.track_number}' if self.track else '',
```

Comment:
Perhaps writing `Track undefined` (perhaps in Hebrew?) would be preferred than the blank string

Reply:
changed to "לא הוגדר מסלול"